docs/en/pid-eaa-issuance.rst

```diff
@@ -240,7 +238,7 @@ The JWS payload of the request object is represented below:
 
 .. note::
 
-    **PID Credential Schema and Status registration:** The PID/(Q)EAA Issuer MUST register all the issued credentials for their later revocation, if needed. 
+    **PID Credential Schema and Status registration:** The PID/(Q)EAA Issuer MUST register all the issued credentials for their later revocation. 
```

Comment:
```suggestion
    **PID/(Q)EAA Credential Schema and Status registration:** The PID/(Q)EAA Issuer MUST register all the issued credentials for their later revocation. 
```

Reply:
We might remove this and deal with it in a new section on credential lifecycle, @peppelinux  what do you think?